File: backend/src/models/postgreSQL/contact.model.js
export default function(sequelize, Sequelize) {
    const Contact = sequelize.define("contact", {
      contactId: {
        type: Sequelize.INTEGER,
        autoIncrement: true,
        primaryKey: true
      },
      firstName: {
        type: Sequelize.STRING,
        required: true
      },
      lastName: {
        type: Sequelize.STRING,
        required: true
      },
      twitter: {
        type: Sequelize.STRING
      },
      avatar: {
        type: Sequelize.STRING
      },
      notes: {
        type: Sequelize.STRING
      },
      favorite: {
        type: Sequelize.BOOLEAN
      }
    });
  
    return Contact;
  };

File: AspNetBackend/AspNetBackend/Models/ContactModel.cs
using System.ComponentModel.DataAnnotations;

namespace AspNetBackend.Models
{
    public class ContactModel
    {
        [Key]
        public int ContactId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Twitter { get; set; }
        public string Avatar { get; set; }
        public string Notes { get; set; }
        public bool Favorite { get; set; }
    }
}
